.github/workflows/release_dev.yml

```
@@ -90,6 +90,44 @@ jobs:
           echo "tag_name=$TAG_NAME" >> $GITHUB_OUTPUT
           echo "build_date=$DATE_TAG" >> $GITHUB_OUTPUT
 
+      - name: Cleanup old releases (debug mode)
+        run: |
+          set -euo pipefail
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)